dora/integration/jnifuse/fs/src/main/java/alluxio/jnifuse/utils/Environment.java

```diff
@@ -124,4 +132,45 @@ public static String getJniLibraryExtension() {
     }
     return isMac() ? ".jnilib" : ".so";
   }
+
+
+  public static Pair<Integer, String> runCommandAndCapture(String... command)
+      throws IOException, InterruptedException {
+    String[] cmds = new String[command.length];
+    int i = 0;
+    for (String cmd : command) {
+      cmds[i++] = cmd;
+    }
+    Process p = new ProcessBuilder(cmds).start();
+    int exitCode = -1;
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    InputStream in = null;
+    InputStream err = null;
+    OutputStream out = null;
+    try {
+      int c;
+      in = p.getInputStream();
+      while ((c = in.read()) != -1) {
+        bout.write(c);
+      }
+      err = p.getErrorStream();
+      while ((c = err.read()) != -1) {
+        bout.write(c);
+      }
+      out = p.getOutputStream();
+      exitCode = p.waitFor();
+    } finally {
+      Closeable[] closeables = new Closeable[]{in, out, err};
+      for (Closeable c : closeables) {
+        if (c != null) {
+          try {
+            c.close();
+          } catch (Exception e) {
+            // Ignore
+          }
+        }
+      }
+    }
+    return Pair.of(exitCode, bout.toString());
```

Comment:
I prefer to add a class for the result of commandAndCapture,  I have seen quite a few places using a pair for this one.  your call

Reply:
sure will do